addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -379,8 +399,8 @@ def _format_for_frontend(self):
             'id': self.id,
             'answers': [{
                 'id': answer.id,
-                'label': answer.name,
-                'redirectLink': answer.redirect_link,
+                "name": answer.name,
+                "redirect_link": answer.redirect_link,
```

Comment:
I didn't check where `_format_for_frontend` is used, but it should be replaced by `_to_store` as they have the exact same purpose.

Either now or as a separate PR, but now would make the most sense probably.